example/eblob_backend.c

```diff
@@ -976,7 +988,9 @@ static void eblob_backend_cleanup(void *priv)
 {
 	struct eblob_backend_config *c = priv;
 
-	eblob_cleanup(c->eblob);
+	if (c->eblob) {
```

Comment:
There are 2 way to cleanup backend:
  1. by calling `backend_cleanup` from `dnet_backend_callbacks` - unused and can be removed
  2. by calling `cleanup` from `dnet_config_backend`

The last one can be called even if backend is uninitialized for freeing resource allocated while parsing config.